frameworks/Crystal/kemal/shard.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
     version: "0.13.3"
   kemal:
     github: "sdogruyol/kemal"
-    version: "0.18.3"
+    branch: master
```

Comment:
Locking down the version number keeps things stable and documents to users of future benchmarks the differences.

So much is changing in Crystal that it would be nice to know this code will work.
If possible, please lock down to a version and/or